packages/core/core/package.json

```diff
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "ajv": "8.17.1",
-    "core-js": "3.40.0",
+    "express": "4.21.2",
```

Comment:
should not be a dependency, devDependency, only used for types, the import might be misleading